server/django_server/settings.py

```diff
@@ -34,23 +34,38 @@
     os.getenv('ANGULAR_URL', 'http://localhost:4200')
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.TokenAuthentication',
+    ),
+    'DEFAULT_PERMISSIONS_CLASSES': (
+        'rest_framework.permission.IsAuthenticated'
+    ),
+}
+
 # Application definition
 
 INSTALLED_APPS = [
-    'app',
-
+    # Django apps
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
 
+    # Local apps
+    'user_account',
+
+    # 3rd party
     'rest_framework',
+    'rest_framework.authtoken',
```

Comment:
What do you mean by 3rd party?

Reply:
It's just to separate what app we develop internally and what app do we use from outside to organize the INSTALLED APP array in the settings.

Reply:
okay makes sense, but why don't we put the rest framework portion with the rest of django?

Reply:
In order to not get mixed up with what Django comes with and what we installed in order to remove them if we decide to not use them anymore